kuma/javascript/src/payments/feedback-form.jsx

```diff
@@ -0,0 +1,106 @@
+// @flow
+import * as React from 'react';
+import { gettext, Interpolated } from '../l10n.js';
+import { getCookie } from '../utils.js';
+
+export const FEEDBACK_URL = '/payments/feedback';
+export const MIN_STRING_LENGTH = 10;
+
+const FeedbackForm = (): React.Node => {
+    const [feedback, setFeedback] = React.useState<string>('');
+    const [status, setStatus] = React.useState<'success' | 'idle'>('idle');
+    const [error, setError] = React.useState<React.Node | null>(null);
+
+    const handleChange = (event: SyntheticInputEvent<HTMLInputElement>) => {
+        const { value } = event.target;
+        setFeedback(value);
+    };
+
+    const handleSubmit = (event: SyntheticEvent<HTMLFormElement>) => {
+        event.preventDefault();
+        const trimmedFeedback = feedback.trim();
+
+        // Feedback should be greater than 10 characters
+        if (trimmedFeedback.length < MIN_STRING_LENGTH) {
+            setError(
+                `To ensure more constructive feedback, a minimum of ${MIN_STRING_LENGTH} characters is required.`
+            );
+            return false;
+        }
+
+        fetch(FEEDBACK_URL, {
+            method: 'POST',
+            body: JSON.stringify({ feedback: trimmedFeedback }),
+            headers: {
+                'Content-Type': 'application/json',
+                'X-CSRFToken': getCookie('csrftoken')
+            }
+        })
+            .then(res => {
+                // Remove focus from button or input
+                if (document.activeElement) {
+                    document.activeElement.blur();
+                }
+
+                if (!res.ok) {
+                    throw new Error(
+                        'Request (POST) to /api/v1/payments/feedback failed'
+                    );
+                }
+                return res;
+            })
+            .then(() => {
+                // Clear form, show thank you message
+                setFeedback('');
+                setError('');
+                setStatus('success');
+            })
+            .catch(() => {
+                setError(
+                    <Interpolated
+                        id={gettext(
+                            "We're sorry, something went wrong. Please try again or send your feedback to <emailLink />."
+                        )}
+                        emailLink={
+                            <a
+                                target="_blank"
+                                rel="noopener noreferrer"
+                                href="mailto:mdn-support@mozilla.com"
```

Comment:
@schalkneethling What's the latest in terms of getting the value of `settings/common.py`'s `CONTRIBUTION_SUPPORT_EMAIL` into the React code?
Honestly, we can wait with this we work on https://github.com/mdn/kuma/issues/6654 but would you mind @mindy putting in a code comment here that mentions something like:
```
/* TODO: not hardcode the email here in code. See https://github.com/mdn/kuma/issues/6654 */
```

Reply:
Added the comment!

Reply:
The value lives here:
https://github.com/mdn/kuma/blob/cf6743f5d903e0e0c67f0a9ebcb2a489667a533f/kuma/settings/common.py#L1739

To get at that in React, add the following to `jinja2/includes/config.html`
Perhaps here: https://github.com/mdn/kuma/blob/cf6743f5d903e0e0c67f0a9ebcb2a489667a533f/jinja2/includes/config.html#L26

```
win.mdn.contributionSupportEmail = '{{ settings. CONTRIBUTION_SUPPORT_EMAIL }}';
```

you can then access it here as:

```
window.mdn.contributionSupportEmail;
```

Of course, this will not work in the SSR world but, this code will be reached on a form submit so, we will be in browser land.

Reply:
Thank you for these instructions!